packages/flame_studio/lib/src/widgets/toolbar/flame_studio_toolbar.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flame_studio/src/core/theme.dart';
+import 'package:flame_studio/src/widgets/toolbar/pause_button.dart';
+import 'package:flame_studio/src/widgets/toolbar/start_button.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+final toolbarHeightProvider = Provider((ref) => 28.0);
+
+class FlameStudioToolbar extends ConsumerWidget {
+  const FlameStudioToolbar({super.key});
+
+  /// List of widgets to show on the left side of the toolbar.
+  static final List<Widget> left = [];
+
+  /// List of widgets to show in the center of the toolbar.
+  static final List<Widget> middle = [
+    const StartButton(),
+    const PauseButton(),
+  ];
+
+  /// List of widgets to show on the right side of the toolbar.
+  static final List<Widget> right = [];
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final height = ref.watch(toolbarHeightProvider); // settings.toolbarHeight;
```

Comment:
```suggestion
    final height = ref.watch(toolbarHeightProvider);
```
Or was that intentionally left there?

Reply:
accident